pkg/controller/seed-controller-manager/kubernetes/address.go

```diff
@@ -30,15 +31,15 @@ import (
 func (r *Reconciler) syncAddress(ctx context.Context, log *zap.SugaredLogger, cluster *kubermaticv1.Cluster, seed *kubermaticv1.Seed) error {
 	var err error
 	// TODO(mrIncompetent): The token should be moved out of Address. But maybe we rather implement another auth-handling? Like openid-connect?
```

Comment:
what about this TODO?

Do we still want to do it? Maybe it would be a good time now then. Or no point so the TODO can be removed?

Reply:
I am not touching this part. This TODO is ancient and you know the old saying: Don't fix what ain't broken.